src/components/Canvas.tsx

```diff
@@ -17,6 +19,7 @@ type Props = {
 const Canvas: React.VFC<Props> = (props) => {
   const { stageRef, lines, setLines } = props;
   const [lineWidth, setLineWidth] = useState(5);
+  const [lineColor, setLineColor] = useState("black");
```

Comment:
ローカルで動作確認しました。
線を書いたらコンソールに以下のようにメッセージが表示されました。
```
The specified value "black" does not conform to the required format.  The format is "#rrggbb" where rr, gg, bb are two-digit hexadecimal numbers.
```
以下のように変更すると解消されました。

```suggestion
  const [lineColor, setLineColor] = useState("#000000");
```